images/7.5 rev. 150212/windowsservercore-1803/sitecore/Sitecore/Website/App_Config/ConnectionStrings.config

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<connectionStrings>
+  <!-- 
+    Sitecore connection strings.
+    All database connections for Sitecore are configured here.
+  -->
+  <add name="core" connectionString="user id=sa;password=HASH-epsom-sunset-cost7!;Data Source=sql;Database=Sitecore.Core" />
+  <add name="master" connectionString="user id=sa;password=HASH-epsom-sunset-cost7!;Data Source=sql;Database=Sitecore.Master" />
+  <add name="web" connectionString="user id=sa;password=HASH-epsom-sunset-cost7!;Data Source=sql;Database=Sitecore.Web" />
+  <add name="analytics" connectionString="mongodb://root:roman@mongo/Sitecore_analytics" />
```

Comment:
Hi @rdissertori, I experimented with mongo a bit. I believe there should be no `roman` as password here. Wouldn't it be better to use mongodb without authentication? (for dev purposes of course).

Reply:
sure, can do - I will change this in all the files (including docker compose ones).
I will update this PR in about 1-2hours